go/host/db/hostdb.go

```diff
@@ -38,25 +41,42 @@ type DB struct {
 	blockReads  gethmetrics.Gauge
 }
 
+func (db *DB) Start() error {
+	return nil
+}
+
 // Stop is especially important for graceful shutdown of LevelDB as it may flush data to disk that is currently in cache
-func (db *DB) Stop() error {
+func (db *DB) Stop() {
```

Comment:
We should prolly surface these errors and handle them at the stop handler at the highest level.